ex3/ex3.c

```diff
@@ -5,10 +5,34 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
-int main(int argc, char* argv[])
+int main(int argc, char *argv[])
 {
-    // Your code here
+    pid_t FPID = fork();
+    pid_t PID = getpid();
+    printf("Starting... %d", PID);
+    pid_t PPID = getppid();
+
+    FILE *fp;
+    char file[] = "text.txt";
```

Comment:
File does not exist

Reply:
Very sorry for the confusion, the argument "a" used in the subsequent fopen(...) call allows the .txt to be generated on an as needed basis overwriting the previous output for clarity.

Reply:
Ah, you're right! My system wasn't allowing the file to be made with the warnings from fprintf, but it does work as expected :)